lib/secret-storage/nondumpable-allocator.c

```diff
@@ -54,37 +59,32 @@ _mmap(gsize len)
   gpointer area = mmap(NULL, len, PROT_READ|PROT_WRITE, MAP_ANONYMOUS|MAP_PRIVATE, -1, 0);
   if (!area)
     {
-      if (logger_fatal)
-        {
-          char reason[32] = { 0 };
-          snprintf(reason, sizeof(reason), "len: %"G_GSIZE_FORMAT", errno: %d\n", len, errno);
-          logger_fatal("secret storage: cannot mmap buffer", reason);
-        }
+      char reason[32] = { 0 };
```

Comment:
Thanks for pointing this out, this was an unintentional side effect from my side. I would leave it as it is now, because it has minor effect on the runtime. Later on, this can be addressed with a more sophisticated logger library implementation.